src/test/run-make/sgx-process-termination/Makefile

```diff
@@ -0,0 +1,8 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# only-sgx
```

Comment:
As discussed, I think this should run on all platforms?

Reply:
Some platforms may need a platform-specific way of running the code (see wasm-* or thumb-* tests in run-make for example), do you want to have platform-specific code here to run the test for those?

Reply:
I think we should just add `ignore` statements for platforms that have special requirements. Everything that can just run with `cargo run` should get tested.